src/indexer/repository/transfer.ts

```diff
@@ -55,6 +54,13 @@ class TransferRepository {
         },
       },
       timestamp: decodedLog.timestamp,
+      account: {
+        create: {
```

Comment:
What happens if the account already exists?

Reply:
We need to check for the account before insertion. I have added this already but I will hold the push until we merge substrate indexing, because upon rebasing, this PR is going to have changes on indexing logic as a result of the schema changes